community_tasks/ger_rag_eval.py

```diff
@@ -0,0 +1,224 @@
+# MIT License
+
+# Copyright (c) 2024 The HuggingFace Team
+# Copyright (c) 2024 Philip May, Deutsche Telekom AG
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+# ruff: noqa: F405, F403, F401
+"""
+Custom evaluation tasks for lighteval. Copy this file and complete it with the info for your task.
+
+This file generally create just a TASKS_TABLE and TASKS_GROUPS which are then imported by LightEval.
+
+Author:
+"""
+import numpy as np
+from aenum import extend_enum
+
+from lighteval.metrics import Metrics
+from lighteval.metrics.metrics import SampleLevelMetric
+from lighteval.metrics.utils import MetricCategory, MetricUseCase
+from lighteval.tasks.lighteval_task import LightevalTaskConfig
+from lighteval.tasks.requests import Doc
+from lighteval.tasks.tasks_prompt_formatting import LETTER_INDICES
+
+
+task1 = LightevalTaskConfig(
+    name="ger_rag_eval_task1",
```

Comment:
make the task names more explicit.
From the doc, something like
`ger_rag_eval_choose_the_question`, 
`ger_rag_eval_choose_the_context`,
`ger_rag_eval_answer_is_correct`,
`ger_rag_eval_is_answerable`.

Same for your prompt_functions

Reply:
Edit: just checked, and since the subsets are indeed task1 to 4, you can keep them, but you need to add a bit of doc. (and please use `german` not `ger`)

Reply:
done